packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -95,6 +118,14 @@ abstract class PortfolioBase extends cdk.Resource implements IPortfolio {
     });
   }
 
+  public addTagOptions(tagOption: TagOption) {
+    AssociationManager.associateTagOption(this, this.portfolioId, tagOption);
+  }
+
+  public allowTagUpdates(product: IProduct, options: TagUpdatesOptions = {}) {
+    AssociationManager.addResourceUpdateConstraint(this, product, options);
```

Comment:
Why this difference in names between `allowTagUpdates` and `addResourceUpdateConstraint`?

Reply:
originally we wanted to have more fluid language for what was going on.  But I am okay with not doing that anymore and just sticking to the pre existing naming conventions. Moving everything to`resourceUpdateConstraint`